helios-core/src/main/kotlin/helios/core/helios.kt

```diff
@@ -76,8 +76,9 @@ sealed class Json {
       is JsBoolean -> ifJsBoolean(this)
     }
 
-  fun add(key: String, value: Json): JsObject =
-    JsObject(hashMapOf(key to value))
+  fun add(key: String, value: Json): Json = this.asJsObject()
```

Comment:
Is the `this` really necessary here?

Reply:
`add` should be moved to `JsObject` as already discussed.

Having an operator on `Json` that only works for `JsObject` is not the kind of API we want to promote in a public API.